tests/integration/test_openstack_builder.py

```diff
@@ -99,6 +99,7 @@ def image_ids_fixture(
         image_config=config.ImageConfig(
             arch=image_config.arch,
             base=config.BaseImage.from_str(image_config.image),
+            microk8s="1.31-strict/stable",
```

Comment:
Just out of curiosity, why use different MicroK8s version in unit and integration tests?

Reply:
No particular reason, diversifying different versions :D